group1/comm/static/comm_tool.js

```diff
@@ -24,7 +24,15 @@ function scroll_messages_into_view() {
     }
 }
 
-
+function startVideoChat(){
```

Comment:
The formatting in some places in this file is better than others.  Would you mind formatting this function so that it is more consistent with some of the better looking code in this file?  There can be a space after the closing paren and before the opening curly brace on line 27.  A space between "if" and the opening paren on line 29.  A space between the closing paren and opening curly brace on line 29.  Spaces between the closing curly brace, "else" and the opening curly brace on line 32.